examples/echobot2.py

```diff
@@ -26,7 +26,7 @@
 logger = logging.getLogger(__name__)
 
 
-# Define a few command handlers. These usually take the two arguments bot and
+# Define a few command handlers. These usually take the two arguments, bot and
```

Comment:
> If he removes that comma there's nothing to merge...

@Eldinnie look at the whole commit, the other change is replacing `you` with `user` on line 67.